src/GafferArnold/IECoreArnoldPreview/Renderer.cpp

```diff
@@ -653,9 +653,9 @@ class ArnoldAttributes : public IECoreScenePreview::Renderer::AttributesInterfac
 			else if( objectType == IECore::ExternalProceduralTypeId )
 			{
 				const IECore::ExternalProcedural *procedural = static_cast<const IECore::ExternalProcedural *>( object );
-				if( const IECore::StringData *nodeType = procedural->parameters()->member<const IECore::StringData>( "ai:nodeType" ) )
+				if( procedural->getFileName() == "volume" )
```

Comment:
Don't suppose you could make a super-quick PR for Cortex master, adding a todo to rename the filename property to account for the usage in Arnold? I'll do the actual work when I'm doing some other refactoring later on...

Reply:
Will do this when I make this change in Cortex master.